### ISSUE TITLE
GoKarts, In race mode, front runner starts second lap.

### ISSUE THREAD

**Issue by conartist6** [OpenRCT2/OpenRCT2#7390]
**OS:** Mac OS 10.13.3
**Version:** 0.1.3-HEAD
**Commit/Build:** bb42319 (DEBUG)

In the Karts & Coasters the King Karts ride is a very long Go Karts. It appears to be bugged, such that the lap leader will proceed to do a lap 2. Since the track is so long, this greatly reduces the profit from the ride and makes the queue move super slow such that my guests start to hate my park.

- [UNSURE] Reproducible in RCT2 (vanilla)?
- [NO] Specific to multiplayer?

**Steps to reproduce:**
1. Open Save File
2. Watch King Karts

**Screenshots / Video:** 
<img width="743" alt="screen shot 2018-04-08 at 11 08 02 am" src="https://user-images.githubusercontent.com/540777/38470820-268d3188-3b1d-11e8-8f4d-97acb09e368d.png">

**Save game:**
[Karts & Coasters.txt](https://github.com/OpenRCT2/OpenRCT2/files/1887739/Karts.Coasters.txt)

**Comment by Broxzier** [comment]
This happens to any Go-Karts ride using race mode; the winner gets to do a victory lap. An option to disable the victory lap would be nice.

Perhaps we could abuse one of the existing ride flags for this, until we move over to our own save format?

**Comment by Gymnasiast** [comment]
The winner is the one who passes the finish first. That's the definition of a race. If you don't want a race, pick 'Continuous Circuit mode', rather than 'Race mode'.

**Comment by conartist6** [comment]
Makes sense, I just figured that the finish was the tail end of the station platform not the front of it.

**Comment by qwertychouskie** [comment]
It would probably make sense to make the finish at the back of the platform.  Or maybe make it an option (Race mode with victory lap).
